tests/test_solvers/test_harbor_solver.py

```
@@ -386,6 +387,164 @@ def test_exposes_all_base_environment_public_attrs(self, tmp_path):
         assert not missing, f"SandboxEnvironmentAdapter missing public attrs: {missing}"
 
 
+class TestConfigureTerminusTokenizer:
+    """Tokenizer is fetched/registered only for terminus-2; failures fail fast."""
+
+    @pytest.fixture(autouse=True)
+    def _reset_state(self, monkeypatch):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_reset_state`

Add return type annotation: `None`

(ANN202)